addon/components/grid-stack-item.js

```diff
@@ -16,15 +16,15 @@
  * Full list of options:
  *   https://github.com/troolee/gridstack.js/tree/master/doc#item-attributes
  */
-import Ember from 'ember';
+import { dasherize } from '@ember/string';
```

Comment:
Can we update all the changed files with ©️s to 2018?

Reply:
Done